app/src/main/java/org/onosproject/ngsdn/tutorial/L2BridgingComponent.java

```diff
@@ -210,7 +210,7 @@ private void insertMulticastFlowRules(DeviceId deviceId) {
         // Modify P4Runtime entity names to match content of P4Info file (look
         // for the fully qualified name of tables, match fields, and actions.
         // ---- START SOLUTION ----
-        // Match ARP request - Match exactly FF:FF:FF:FF:FF
+        // Match ARP request - Match exactly FF:FF:FF:FF:FF:FF
```

Comment:
Can you please update the corresponding solution file?

Reply:
Solution code was OK, just the text